direct/io-pubsub/src/main/java/cz/o2/proxima/direct/pubsub/PubSubReader.java

```diff
@@ -79,7 +82,6 @@
 @Stable
 @Slf4j
 class PubSubReader extends AbstractStorage implements CommitLogReader {
```

Comment:
We should call `onIdle` here somewhere.

Reply:
Or at least create issue that PubSubReader is not fulfilling the complete contract of watermark estimator.

Reply:
I have tried to find some easy solution but without success. I dont want to implement it now, because I don't have access to PubSub. I would prefer create issue for that.

Reply:
Created https://github.com/O2-Czech-Republic/proxima-platform/issues/154